### PR TITLE
Fix/tuple not valid json

### LINKED ISSUE #4: Tuple should not be used
This is marked as a "tuple" type

https://github.com/sublimelsp/lsp-python-types/blob/58beea8f2ea918eb9262d2f61e078b85e1ee88f1/lsprotocol/lsp.json#L10102-L10114

but since the data is transported as JSON and there is no concept of tuple in JSON, I think that generated type needs to use a list.

### PR DESCRIPTION
**Pull request by predragnikolic**
closes #4

### CODE REVIEW

File: lsp_types.py [hunk 1]
```diff
@@ -4124,7 +4124,7 @@ class DiagnosticRelatedInformation(TypedDict):
 class ParameterInformation(TypedDict):
     """ Represents a parameter of a callable-signature. A parameter can
     have a label and a doc-comment. """
-    label: Union[str, Tuple[Uint, Uint]]
+    label: Union[str, List[Union[Uint, Uint]]]
```
Comment (rchl) [inline lsp_types.py:4127]: Union with two `Uint` doesn't quite make sense. I think it's meant to be a list of two Uint items.

Comment (predragnikolic) [reply lsp_types.py:4127]: > I think it's meant to be a list of two Uint items.

I think it means a list could contain the following types Uint, Uint, (which doesn't make sense tor repeat, but it is a valid syntax)

```
x: ParameterInformation = {
    'label': [1, 2, 3] # this is valid
}
```

Comment (rchl) [reply lsp_types.py:4127]: It might be valid but seems equivalent to just `List[Unit]` and doesn't add any restrictions beyond that variant.

Comment (rchl) [reply lsp_types.py:4127]: But I suppose I should let it go as it's technically valid.

It doesn't put constrains on number or order of list items but I guess there might not be a solution for that in Python lists. I suppose python expects Tuples for this case.

**Review by rchl — APPROVED** [review]

**Review by rchl — APPROVED** [review]

File: lsp_types.py (hunk 1 above)
Comment (predragnikolic) [reply lsp_types.py]: I guess I would rather revert [689eaf0](https://github.com/sublimelsp/lsp-python-types/pull/6/commits/689eaf0b026b29fb66d96f49e7bf52643ab0a533)

Than to add more code to handle this case:

/home/runner/work/lsp-python-types/lsp-python-types/lsp_types.py:4127:28 - error: Union requires two or more type arguments

**Pull request merged**